content/en/data_observability/_index.md

```diff
@@ -2,28 +2,22 @@
 title: Data Observability
 description: "Monitor data quality, performance, and cost with Data Observability to detect anomalies, analyze data lineage, and prevent issues affecting downstream systems."
 further_reading:
-  - link: '/data_observability/datasets'
+  - link: '/data_observability/data_quality'
```

Comment:
I found it a little confusing when the link text didn't match the title of the page this went to. Should it go here instead?
```suggestion
  - link: '/data_observability/quality_monitoring/'
```